Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/EntryDelegate.h

```diff
@@ -52,5 +52,17 @@ namespace AzToolsFramework
             //! Draw a thumbnail and return its width
             int DrawThumbnail(QPainter* painter, const QPoint& point, const QSize& size, Thumbnailer::SharedThumbnailKey thumbnailKey) const;
         };
+
+        class SearchEntryDelegate
```

Comment:
suggestion: This could use an API comment to explain what it's used for and where